movement/movement_config.h

```diff
@@ -30,6 +30,7 @@
 const watch_face_t watch_faces[] = {
     simple_clock_face,
     world_clock_face,
+    alarm_face,
```

Comment:
Let's just make sure to remove this before merging so as not to change the default build :)

Reply:
Sure